Langs/0401.lang

```diff
@@ -0,0 +1,59 @@
+[MetaData]
+Language=العربية - المملكة المغربية
+Translator=Kadantte - [ Jae Jin ]
+Version=2.2.5.0
+[Strings]
+Donate=تبرع
+Check for Updates=البحث عن التحديثات
+Your Windows 11 Compatibility Results are Below=نتائج توافقية حاسوبك مع ويندوز 11 بالأسفل
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=للوصول إلينا WhyNotWin11
+Results Based on Currently Known Requirements!=النتائج مبنية على المتطلبات المعروفة حاليا
+Translation by=المترجم
+Architecture (CPU + OS)=معمارية المعالج ونظام التشغيل
+Boot Method=نوعية الإقلاع
+CPU Compatibility=توافقية المعالج
+CPU Core Count=عدد أنوية المعالج
```

Comment:
plural of نواة is نوى

Reply:
It can be أنوية

Please refer to wiki: https://ar.wikipedia.org/wiki/%D8%A8%D8%A7%D9%88%D8%B17#:~:text=%D8%B9%D8%AF%D8%AF%20%D8%A7%D9%84%D8%A3%D9%86%D9%88%D9%8A%D8%A9%20%3A%2D%20%D8%AA%D8%AD%D8%AA%D9%88%D9%8A%20%D9%83%D9%84%20%D8%B4%D8%B1%D9%8A%D8%AD%D8%A9,MultiThreading%20)%20%D9%88%D9%87%D9%8A%20%D8%AA%D8%B3%D9%85%D9%89%20%D8%A3%D9%86%D9%88%D9%8A%D8%A9%20%D8%AA%D8%AE%D9%8A%D9%84%D9%8A%D8%A9.

Reply:
please consider that there is no effective moderation in the arabic wikipedia, you gotta a check a dictionnary ;

https://www.almaany.com/ar/dict/ar-ar/%D9%86%D9%88%D8%A7%D8%A9/